ProjectSidewalk/SidewalkWebpage
# #2697 Add Validation Info to Gallery Cards

Resolves #2697

Adds validation information to the Gallery Page in each card and in modals, using a thumbs up and a thumbs down image along with a count for each, indicating the agree count and the disagree count of each attribute. 
- Oh hover of either the thumbs up or thumbs down image, a tooltip appears with the label "Agree" or "Disagree,"  respectively. These labels change based on the language the page is being displayed in.
- The original design showed "Agree" and "Disagree" above each image, rather than in a tooltip. However, because the Spanish translations are so long, Mikey and I decided it would be better to display these labels in tooltips. Some other options we could explore include finding shorter translations for the terms "Agree" and "Disagree."
- The thumbs up and a thumbs down images are stored as PNGs.

#### Before/After screenshots
##### Before:
Cards:
<img width="390" alt="Screen Shot 2023-02-10 at 11 17 58 AM" src="https://user-images.githubusercontent.com/50668250/218182047-678e803b-7d8c-4183-86e0-d070996fc6bd.png">

Modal:
<img width="1440" alt="Screen Shot 2023-02-10 at 11 18 09 AM" src="https://user-images.githubusercontent.com/50668250/218182073-f0cdf4c0-8c94-4937-8ee0-5f4aa99d4ac6.png">


##### After:
Cards:
<img width="300" alt="Screen Shot 2023-02-10 at 10 55 02 AM" src="https://user-images.githubusercontent.com/50668250/218182124-ee996f10-82f2-47b9-9ccc-675527cd2953.png">   <img width="300" alt="Screen Shot 2023-02-10 at 10 55 11 AM" src="https://user-images.githubusercontent.com/50668250/218182149-f9e24a19-99b6-4996-b4ca-492a6f8423f1.png">
<img width="300" alt="Screen Shot 2023-02-10 at 10 59 37 AM" src="https://user-images.githubusercontent.com/50668250/218182181-972a7ec0-a5b5-4ee6-a584-f923bfedc7ef.png">   <img width="300" alt="Screen Shot 2023-02-10 at 11 01 06 AM" src="https://user-images.githubusercontent.com/50668250/218182193-559670f7-cdd0-4640-bece-a3b91a24e630.png">
<img width="300" alt="Screen Shot 2023-02-10 at 11 01 22 AM" src="https://user-images.githubusercontent.com/50668250/218182204-56e2bf6f-9b41-469f-9135-4c1b3c513d6c.png">

Modal:
<img width="1440" alt="Screen Shot 2023-02-10 at 10 55 25 AM" src="https://user-images.githubusercontent.com/50668250/218182161-055453df-023e-457a-9078-6cf0b063d356.png">

#### Testing instructions
1. Run the branch locally
2. Navigate to the Gallery page
3. The validation info should be visible on each card
4. Hover over the agree and disagree displays to view the Agree/Disagree tooltip label of each
5. Click on a card to open a modal. Here, you should also be able to see the validation info. Again, hover over the agree and disagree displays to view the tooltip label of each
